ENOCK JEANNICK OMAR bug41:need to be submitted by whom information. t…

Jeannick94 opened the pull request:
The system shows the information of the person that submitted the assignment. In submissions.php, it shows the name and the student Id of the person that submitted the assignment on behalf of the group.

Jeannick94 committed:
ENOCK JEANNICK OMAR bug41:need to be submitted by whom information. the system shows the information of the person that submit the assignement.

enockkays committed:
Update Script.php

Corrected the wrong spelling which was spelt as "happend" instead of "Happened".

spm2020spring commented on Submissions.php:
```diff
@@ -163,14 +164,15 @@
                     $submitted_by= $names."(".$student_id.")";
                 } else {
-                    $submitted_by="<i>(GROUP)</i> $groupname" ;
+                    $submitted_by="$names $submitted_std <i>(GROUP)</i> $groupname ";
```
@Jeannick94 @enockkays 
Wher is `$submitted_std` defined?

-Hui

Jeannick94 replied:
@spm2020spring 
It's defined on line 163 of the submissions file

Jeannick94 committed:
Update Submissions.php

 We replace "$names $submitted_std <i>(GROUP)</i> $groupname " with "$names ($submitted_std) for group $groupname"?

We made variable names more descriptive by  replacing submitted_std with submitter_student_number and also names with student_name
